pkg/config/config.go

```diff
@@ -28,6 +28,7 @@ func init() {
 	flag.StringVar(&Flags.ServiceAccountTokenPath, "service-account-token-path", "", "optionally override the default token path")
 	flag.StringVar(&Flags.MetricsAddr, "metrics-addr", "0.0.0.0:8081", "address to serve Prometheus metrics on")
 	flag.StringVar(&Flags.ProbeAddr, "probe-addr", "0.0.0.0:8080", "address to serve readiness/liveness probes on")
+	flag.StringVar(&Flags.OperatorDeploy, "operator-deploy", "app-routing-operator", "name of the operator's k8s deployment")
```

Comment:
nit: OperatorDeployment :) if you have a reason to use this name instead I'd be interested as to why

Reply:
Good call - I'll use the full name instead of the shorthand